private-templates-service/src/models/Template.ts

```diff
@@ -0,0 +1,14 @@
+import mongoose from 'mongoose';
+
+const templateSchema = new mongoose.Schema({
```

Comment:
I think there should be another layer of abstraction here, as we don't want to lock the user to mongo

Reply:
Yep, we're working on a database adapter that will work for different databases - I've used a local mongo db for now just to set up the endpoints